Model/ConfigurationFactory.php

```diff
@@ -40,9 +40,15 @@ public function __construct(
      */
     public function create(int $storeId)
     {
-        return $this->objectManager->create(\Drip\Connect\Model\Configuration::class, ['storeId' => $storeId]);
+        $websiteId = $this->storeManager->getStore($storeId)->getWebsiteId();
+		return $this->createFromWebsiteId($websiteId);
```

Comment:
Some indentation goofiness here. I'm realizing that the entire file looks four-space indented without tabs. The new code is all tab indented. I wonder whether that's the issue, that you need to switch your editor to spaces…

Reply:
Switched to all spaces and I think things look good.